todo

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+source $HOME/.todorc
+
+TODO_PATH="${TODO_PATH:=$HOME/.todo}"
+add_task(){
+
+	temp=$(cat $TODO_PATH/id.txt)
```

Comment:
When the `id.txt` file doesn't exist, the message below shouldn't be displayed to the user

```
cat: /home/x/.todo/id.txt: No such file or directory
```